crates/ai/src/models.rs

```diff
@@ -14,3 +17,12 @@ pub trait LanguageModel {
     ) -> anyhow::Result<String>;
     fn capacity(&self) -> anyhow::Result<usize>;
 }
+
+#[derive(Clone, Copy, Debug, Default, Serialize, Deserialize, PartialEq, JsonSchema)]
+pub enum ModelEndpoint {
+    #[serde(rename = "OPENAI")]
```

Comment:
Is this all-uppercase serialized representation something imposed on us by an external API?

If not, these should be snake_cased (and we can do it with a `#[serde(rename_all = "snake_case")]` attribute on the enum itself

Reply:
It doesn't have to be uppercased